MODULE.bazel

```diff
@@ -0,0 +1,11 @@
+bazel_dep(name = "abseil-cpp", version = "20230802.0", repo_name = "com_google_absl")
+bazel_dep(name = "abseil-py", version = "1.4.0", repo_name = "com_google_absl_py")
+bazel_dep(name = "bazel_skylib", version = "1.5.0")
+bazel_dep(name = "grpc", version = "1.48.1")
```

Comment:
This change removes the usage dependency on grpc proto rules.

Reply:
Good catch, thanks!